types/highland/index.d.ts

```diff
@@ -899,6 +921,7 @@ declare namespace Highland {
 		 * @param {Number} n - integer representing number of values to read from source
 		 * @api public
 		 */
+
```

Comment:
Could you move this new line so as to be on 915? (IE, between the end of the splitBy definition and the beginning of the comment describing the take definition)